default.xml

```diff
@@ -22,7 +22,7 @@
   <project path="hybris/mer-kernel-check" name="mer-hybris/mer-kernel-check" revision="master" />
   <project path="external/droidmedia" name="sailfishos/droidmedia" revision="master" />
 
-  <project path="android" name="CyanogenMod/android" />
+  <project path="android" name="mer-hybris/android" revision="hybris-12.1" />
```

Comment:
Other (older) branches don't sync the manifest, should we nuke it here too? I.e. it's always accessible through .repo/manfests/ anyway